packages/react/src/components/TimePicker/TimePicker.types.ts

```diff
@@ -0,0 +1,62 @@
+import { IComboBoxProps } from '../../ComboBox';
+
+/**
+ * Range of start and end hours to be shown in the TimePicker.
+ */
+export interface ITimeRange {
+  /** Start hour (inclusive) for the time range, 0-23 */
+  start: number;
+  /** End hour (exclusive) for the time range, 0-23 */
+  end: number;
+}
+
+/**
+ * Localized strings to use in the TimePicker
+ */
+export interface ITimePickerStrings {
+  /** Error message to render for TextField if input parsing fails. */
```

Comment:
Nit, it's not a `TextField`, so to avoid confusion the message should be more generic.
```suggestion
  /** Error message to render below the field if input parsing fails. */
```

Reply:
Resolved by updating comment to be `/** Error message to render below the field if input parsing fails. */`.